views/question-table.pug

```diff
@@ -29,8 +33,12 @@ block styles
           tbody
             each q in questions
               tr
-                td
+                td.center
                   p
+                    if q.visible
+                      i.material-icons.hiddenEye(id='hiddenEye' + q._id, onClick='visibilitySnackbar()') visibility
```

Comment:
Again 2 things with the same id

Reply:
The ID for these icons is hiddenEye + q._id. They do have the same classes but this is necessary to hide/show these icons depending on if the instructor is in edit/display mode.